compute/admin_guide/vulnerability_management/vm_image_scanning.adoc

```diff
@@ -150,10 +150,9 @@ compute.subnetworks.use
 compute.subnetworks.useExternalIp
 ----
 
-* Verify that your project has a service account with the `Cloud KMS CryptoKey Decrypter` role or equivalent.
-
-* Verify that the **Compute Engine Service Agent** xref:https://cloud.google.com/iam/docs/service-agents[Google-managed service account] ends with `compute-system.iam.gserviceaccount.com`
-
+* Verify that the Compute Engine Service Agent service account in the target image project has the `Cloud KMS CryptoKey Decrypter` role or equivalent. 
++
+This built-in service account ends with `compute-system.iam.gserviceaccount.com`.
```

Comment:
Good, but please at the end of the sentence also add a link for reference to https://cloud.google.com/iam/docs/service-agents.

Reply:
I've added the link again. Thanks for catching that.